testing/test_capabilities.py

```diff
@@ -64,6 +76,22 @@ def capabilities():
     testdir.quick_qa(testfile, passed=1)
 
 
+def test_session_scoped_fixture(testfile_scopes, testdir):
+    testdir.makeconftest(
+        """
+        import pytest
+        @pytest.fixture(scope='session')
+        def chrome_options(chrome_options):
+            chrome_options.add_argument('--headless')
+            return chrome_options
+    """
+    )
+    testdir.quick_qa(testfile_scopes, failed=1)
```

Comment:
@garethmj  Why do you expect this test to fail? You set the scope to session via cmd_line and the derive a fixture form chrome_options fixture which should be session scoped? 

I would expect test cases like this:

cmdline flag set (seee example): 
* session scoped fixture passes
* function scoped fixture fails
cmdline flag not set:
 * session scoped fixture fails
 * function scoped fixture passes
```
        """
        import pytest
        @pytest.mark.fixture(scope="session")
        def my_fixture(<selenium_fixture_where_scope_can_be_determined>):
            return

       # define this in testfile
       def test_stuff(my_fixture):
           assert True
    """
    testdir.quick_qa("--selenium-session-scope", passed=1)

...
        """
        import pytest
        @pytest.mark.fixture(scope="function")
        def my_fixture(<selenium_fixture_where_scope_can_be_determined>):
            return

       # define this in testfile
       def test_stuff(my_fixture):
           assert True
    """
    testdir.quick_qa("--selenium-session-scope",  failed=1)
```

Ideally <selenium_fixture_where_scope_can_be_determined> should be tested with all fixtures where the scope can be determined, so I would also recommend a new file for these scope tests.